ts/Series/AreaRange/AreaRangePoint.ts

```diff
@@ -103,9 +93,21 @@ class AreaRangePoint extends AreaPoint {
 
     public plotLowX?: number;
 
-    public plotX: number = void 0 as any;
+    public plotX!: number;
+
+    public series!: AreaRangeSeries;
 
-    public series: AreaRangeSeries = void 0 as any;
+    /**
+     * Range series only. The high or maximum value for each data point.
+     * @name Highcharts.Point#high
+     * @type {number|undefined}
+     */
+
+    /**
+     * Range series only. The low or minimum value for each data point.
+     * @name Highcharts.Point#low
+     * @type {number|undefined}
+     */
```

Comment:
These do not get picked up, because `tools/gulptasks/jsdoc-classes.js` only references `AreaRangeSeries.ts`.

Reply:
Hmm. So this has never been working then. I can add it to this commit.

Reply:
Sorry for piggy-packing.

Reply:
Now I am even feeling more sorry, because the changes failed. I will take a look with a separate PR after merge, unless you like to check it yourself.